devtools/plugins/desktop/basic/src/WrapperComponent.tsx

```diff
@@ -156,7 +157,12 @@ export const WrapperComponent = ({
     if (dequal(state.plugins[pluginID]?.flow?.data?.data, data)) return;
 
     const newState = produce(state, (draft) => {
-      set(draft, ["plugins", pluginID, "flow", "data", "data"], data);
+      try {
+        dset(draft, ["plugins", pluginID, "flow", "data", "data"], data);
+      } catch {
+        console.error("Error setting the following data: ", data);
```

Comment:
let's create an issue to display those errors in the dev tools panel, instead of the console for easy access in the future. Maybe we can leverage the toast we created for copy to clipboard.